tx_metric/tx_metric.go

```diff
@@ -51,7 +59,12 @@ func (tm *TxMetric) Start() error {
 
 		if bl.Time() <= tm.StopTime {
 			fmt.Printf("Found blocknumber %d | Txs: %d\n", i, bl.Transactions().Len())
-			totalTx += bl.Transactions().Len()
+			numberOfTx := bl.Transactions().Len()
+			if calculateEachMinuteFlag {
+				minuteStats[int(math.Floor(float64(bl.Time()-tm.StartTime)/60.0))] += numberOfTx
```

Comment:
`int(bl.Time()-tm.StartTime)/60` is ok, not need math.Floor

Reply:
Updated